client/my-sites/checkout/src/components/checkout-sidebar-plan-upsell/index.tsx

```diff
@@ -156,16 +156,14 @@ export function CheckoutSidebarPlanUpsell() {
 		<>
 			<PromoCard title={ cardTitle } className="checkout-sidebar-plan-upsell">
 				<div className="checkout-sidebar-plan-upsell__plan-grid">
-					{ isComparisonWithIntroOffer && (
-						<>
-							<div className="checkout-sidebar-plan-upsell__plan-grid-cell"></div>
-							<div className="checkout-sidebar-plan-upsell__plan-grid-cell">
-								<strong>{ __( 'Two-year cost' ) }</strong>
-							</div>
-						</>
-					) }
 					<div className="checkout-sidebar-plan-upsell__plan-grid-cell">
-						{ currentVariant.variantLabel }
+						<strong>{ __( 'Plan' ) }</strong>
+					</div>
+					<div className="checkout-sidebar-plan-upsell__plan-grid-cell">
+						<strong>{ isComparisonWithIntroOffer ? __( 'Two-year cost' ) : __( 'Cost' ) }</strong>
```

Comment:
For some reason 'Two-year cost' doesn't sit well with me as a header. Maybe it is because I interpret this in the same way as the second line item where it also says 'Two-year'. 

But I am not sure what alternative would fit better.

Reply:
I'm also not sure why there is a differentiation between `annual -> biennial` and `monthly -> biennial` here. Wouldn't the second line still be the monthly cost over two years compared to the two-year billing term of the plan?